packages/core/src/RenderingEngine/helpers/setDefaultVolumeVOI.ts

```diff
@@ -187,10 +202,7 @@ async function getVOIFromMinMax(
   let image = cache.getImage(imageId);
 
   if (!imageVolume.referencedImageIds?.length) {
-    // we should ignore the cache here,
-    // since we want to load the image from with the most
-    // recent prescale settings
-    image = await loadAndCacheImage(imageId, { ...options, ignoreCache: true });
+    image = await loadAndCacheImage(imageId, options);
   }
 
   const imageScalarData = image
```

Comment:
Could you revert this part as well? We need it.

Reply:
Sure, I agree with you.